frontend/src/pages/RecipientRecord/components/RecipientSpotlight.js

```diff
@@ -0,0 +1,140 @@
+      try {
+        const response = await getRecipientSpotlight(
+          String(recipientId),
+          String(regionId),
+        );
+
+        // Check if response is valid and has meaningful data (more than just an empty object)
+        const hasValidData = response
+          && response.length > 0
+          && response[0]
+          && Object.keys(response[0]).length > 1;
+
+        if (hasValidData) {
+          setSpotlightData(mappedData(response[0]));
+          setHasResults(true);
+        } else {
+          setSpotlightData([]);
+          setHasResults(false);
+        }
+      } catch (err) {
+        setSpotlightData([]);
+        setHasResults(false);
+      }
+    }
+    fetchRecipientSpotlight();
+    /// setSpotlightData(mappedData(sampleSpotlightData[0] || {}));
+  }, [recipientId, regionId]);
+
+  const hasIndicators = spotlightData.some((indicator) => indicator.value === true);
+
+  const numberOfTrueIndicators = spotlightData
+    .filter((indicator) => indicator.value === true)
+    .length;
+
+  return (
+    <Container paddingX={0} paddingY={0} className="ttahub--recipient-summary">
+      <div className="ttahub-recipient-record--card-header padding-x-3 padding-y-3 margin-bottom-0 margin-top-0">
+        <h2 className="margin-0 padding-0">Recipient spotlight</h2>
+        <p className="margin-0 padding-0 usa-prose">This is the recipient&apos;s current number of priority indicators.</p>
+      </div>
+
+      {hasResults ? (
+        <div className="ttahub-recipient-spotlight-content padding-3 overflow-y-auto">
+          <div className="display-flex flex-align-center">
+            <div className="display-flex flex-column">
+              <div className="display-flex flex-align-center">
+                {hasIndicators
+                  ? <FontAwesomeIcon className="margin-right-1" size="2x" color={colors.error} icon={faCircleExclamation} />
+                  : <FontAwesomeIcon className="margin-right-1" size="2x" color={colors.success} icon={faCircleCheck} />}
+                <h4 className="margin-0 Merriweather">{hasIndicators ? 'Recipient may need prioritized attention' : 'No priority indicators identified'}</h4>
+              </div>
+              <IndicatorCounter count={numberOfTrueIndicators} totalCount={7} />
+            </div>
+          </div>
+
+          <div className="display-flex flex-align-center margin-top-1">
+            <div className="flex-row">
+              <b><p className="usa-prose margin-0 margin-bottom-2">{`${numberOfTrueIndicators} of 7 priority indicators`}</p></b>
+              <div>
+                {
+                spotlightData.map((indicator) => (
+                  <div key={indicator.name} className={`ttahub-recipient-spotlight-content-cell ttahub-recipient-spotlight-content-cell${indicator.value ? '-bad-indicator' : '-good-indicator'} radius-md width-full display-flex flex-align-center margin-bottom-1 padding-x-2 padding-y-1`}>
```

Comment:
Hide the non-applicable indicators (I'm assuming that the x means it doesn't apply, the graph is confusing)

```suggestion
                  <div key={indicator.name} aria-hidden={indicator.value} className={`ttahub-recipient-spotlight-content-cell ttahub-recipient-spotlight-content-cell${indicator.value ? '-bad-indicator' : '-good-indicator'} radius-md width-full display-flex flex-align-center margin-bottom-1 padding-x-2 padding-y-1`}>
```